eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -11,9 +11,9 @@
 # 2. There are no duplicate entries in the external_dependencies.txt file
```

Comment:
Hi, @JimSuplizio , 
Could you please review this file change?
No logic change, just:
* Remote reduantant whitespace. It's removed by Intellij automically.
* Fix the order to 'a, b, c, d.'
* Fix the error in the message.

You can check each commit for easier review.

(cc @saragluna , FYI.)

Reply:
@JimSuplizio ,
Thank you for your response. 🙇 

> These extra spaces are the result of VS Code's formatting

We  can configure VS Code to remove trailing space: https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut